corehq/apps/hqcase/management/commands/reindex_sql_forms_in_domain.py

```diff
@@ -1,5 +1,7 @@
 from django.core.management import BaseCommand
 
+from dimagi.utils.chunked import chunked
```

Comment:
is this necessary?

Reply:
ya that import is missing in that file, found it while searching for something else